modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/GroupUpdateRequest.java

```diff
@@ -304,7 +304,7 @@ private static CompletableFuture<Integer> partitionUpdate(
             long revision,
             MetaStorageManager metaStorageMgr,
             Set<Assignment> currentAssignments,
-            LocalPartitionStateMessageByNode localPartitionStateMessageByNode,
+            @Nullable LocalPartitionStateMessageByNode localPartitionStateMessageByNode,
```

Comment:
Can we simply pass an empty map instead of null, instead of duplicating the calculating logic in `nextAssignment`? I don't like copy-pasted algorithms, they make code harder